### PR TITLE
[Docs][Connector-V2][S3File]Reconstruct the S3File connector document

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/S3File.md [hunk 1]
```diff
@@ -30,89 +22,66 @@ By default, we use 2PC commit to ensure `exactly-once`
...
+To use this connector you need put hadoop-aws-3.1.4.jar and aws-java-sdk-bundle-1.11.271.jar in ${SEATUNNEL_HOME}/lib dir.
 
-Whether custom the filename
+:::
 
-### file_name_expression [string]
+## Data Type Mapping
 
-Only used when `custom_filename` is `true`
+SeaTunnel will write the data into the file in String format according to the SeaTunnel data type and file_format_type,
```
Comment (TaoZex) [inline docs/en/connector-v2/sink/S3File.md]:
```suggestion
SeaTunnel will write the data into the file in String format according to the SeaTunnel data type and file_format_type.
```

**Review by TaoZex — APPROVED** [review]
LGTM
Fixed this typo will be better.

**Comment by ic4y** [comment]
@Hisoka-X We need to resolve the conflicts.

**Comment by zhilinli123** [comment]
fix ci

**Comment by ic4y** [comment]
@Hisoka-X 😆

**Comment by Hisoka-X** [comment]
This is duplicate of #5101

**Pull request closed without merging**